resources/__init__.py

```diff
@@ -38,6 +38,17 @@ def endpoint(*args, **kwargs):
     return endpoint
 
 
+def deprecated(f):
+    """
+    Decorator for deprecated methods
+    """
+    @wraps(f)
+    def endpoint(*args, **kwargs):
+        logger.warning("Used deprecated endpoint: %s", request.path)
```

Comment:
Do we have information about the authorized user at this point? I remember that it was somehow added implicitly but want to confirm.

Reply:
Yes, it is added implicitly here: https://github.com/CERT-Polska/malwarecage/blob/master/core/log.py